build/python/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM python:3
```

Comment:
Is folder name python better or pypi ?

Reply:
I think it's better to follow the upstream docker image naming here. Can easily change later.